src/SFA.DAS.AssessorService.Application.Api.UnitTests/Validators/ExternalApi/Epas/DeleteBatchEpaRequestValidator/WhenDifferentEpao.cs

```diff
@@ -0,0 +1,39 @@
+﻿using FizzWare.NBuilder;
+using FluentAssertions;
+using FluentValidation.Results;
+using NUnit.Framework;
+using SFA.DAS.AssessorService.Api.Types.Models.ExternalApi.Epas;
+using SFA.DAS.AssessorService.Domain.JsonData;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AssessorService.Application.Api.UnitTests.Validators.ExternalApi.Epas.DeleteBatchEpaRequestValidator
+{
+    public class WhenDifferentEpao : DeleteBatchEpaRequestValidatorTestBase
```

Comment:
It's not obvious to me why the data sent to the test fails or doesn't - how can I tell that?

Reply:
Existing business rules state that the EPAO who wishes to modify a certificate must be the same as the one who created the certificate.

Reply:
That's exactly the kind of stuff that isn't being written down anywhere - what a great opportunity to add some comments!